src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: normal;
+  font-size: 600;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-box {
+  display: block;
+  /* flex-direction: row; */
+  align-items: center;
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  /* z-index: 2; */
+  padding-left: 26px;
+  padding-top: 25px;
+  width: 45px;
+  height: 44px;
+  /* padding-right: 26px; */
+  /* padding-bottom: 20px; */
+}
```

```suggestion
.icon {
  position: absolute;
  padding-left: 26px;
  padding-top: 25px;
  width: 45px;
  height: 44px;
}
```
don't forget to remove comments